### PR TITLE
ui: Add link to account role in listview

### PR DESCRIPTION
**Pull request by davidjumani**
### Description

Adds a link in the list view for the role belonging to the account

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

### Screenshots (if appropriate):

Screenshot from 2022-03-07 16-48-47

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by nvazquez — APPROVED** [review]
LGTM

File: ui/src/components/view/ListView.vue [hunk 1]
```diff
@@ -265,6 +265,10 @@
       <router-link v-if="$router.resolve('/zone/' + record.zoneid).route.name !== '404'" :to="{ path: '/zone/' + record.zoneid }">{{ text }}</router-link>
       <span v-else>{{ text }}</span>
     </span>
+    <a slot="rolename" slot-scope="text, record">
+      <router-link v-if="record.roleid" :to="{ path: '/role/' + record.roleid }">{{ text }}</router-link>
```
Comment (rohityadavcloud) [inline ui/src/components/view/ListView.vue:269]: It's probably worth checking if the caller has access to `listRoles` too.

**Review by utchoang — APPROVED** [review]

**Comment by davidjumani** [comment]
@blueorangutan ui

**Comment by davidjumani** [comment]
@blueorangutan ui

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by nvazquez — APPROVED** [review]
LGTM - manually tested ok

**Pull request merged**